xcape/CAPE_CODE_model_lev.f90

```diff
@@ -46,51 +46,39 @@ subroutine loopcape(p3d_in , t3d_in , td3d_in, ps_in , ts_in , tds_in, &
     !f2py threadsafe
     !f2py intent(out) :: cape3d,cin3d,z_out3d,MUlvl3d
 
-    integer, intent(in) :: nk,n2
+    integer, intent(in) :: nk_in,n2
```

Comment:
nk is used in most of the routine as the number of actual levels to loop through, so I have changed this intermediate number of levels (before merging the surface values) to nk_in